cpp/cmake_modules/FindBrotli.cmake

```diff
@@ -65,16 +65,27 @@ if (BROTLI_INCLUDE_DIR AND (PARQUET_MINIMAL_DEPENDENCY OR BROTLI_LIBRARIES))
   set(BROTLI_FOUND TRUE)
   get_filename_component( BROTLI_LIBS ${BROTLI_LIBRARY_ENC} PATH )
   set(BROTLI_LIB_NAME brotli)
-  if (MSVC AND NOT BROTLI_MSVC_STATIC_LIB_SUFFIX)
-    set(BROTLI_MSVC_STATIC_LIB_SUFFIX _static)
+  if (BROTLI_MSVC_STATIC_LIB_SUFFIX)
```

Comment:
Windows builds are still failing because this is set to `_static` in `arrow/cpp/CMakeLists.txt`. I'll submit a PR to your fork to fix the default and update the README in apidoc.